### PR TITLE
Update Headings.yml with SSO, OAuth, Terraform

### PR DESCRIPTION
**Pull request by mgyongyosi**
I'd like to use those in https://github.com/grafana/website/pull/17940

### CODE REVIEW

**Comment by mgyongyosi** [comment]
Moved back to Draft, because I'm gonna add other exceptions probably.

**Comment by jdbaldry** [comment]
Hey bud, any updates on this? Have you found additional terms to add to this PR?

**Comment by mgyongyosi** [comment]
Hey, previously I forgot to link the other PR which we are still working on: https://github.com/grafana/website/pull/17940

I'm gonna reopen this PR when we know all of the exceptions we need for that PR! ;)

**Comment by mgyongyosi** [comment]
@jdbaldry This is ready for review! 🚀

File: vale/Grafana/Headings.yml [hunk 1]
```diff
@@ -24,7 +25,10 @@ exceptions:
   - CPU
   - eBPF
   - Frontend Observability
+  - Generic
```
Comment (jdbaldry) [inline vale/Grafana/Headings.yml:28]: I wonder if _Generic_ should be an exception in of itself. Is it part of a multi-word term that is used in headings?

**Review by jdbaldry — COMMENTED** [review]
Thanks for the contribution!

I've moved the exceptions into our ruleset and regenerated the dictionary from the wordlist.

All the changes except one look perfectly good to me.

File: vale/Grafana/Headings.yml (hunk 1 above)
Comment (mgyongyosi) [reply vale/Grafana/Headings.yml:28]: Yes, `Generic OAuth`, which is the name of the integaration.

Comment (jdbaldry) [reply vale/Grafana/Headings.yml:28]: How about:

```suggestion
  - Generic OAuth
```

**Pull request merged**